installation/rasppi.md

```diff
@@ -79,46 +85,289 @@ Raspbian Lite comes without Java to begin with.
 The following steps refer specifically to Raspberry Pi devices. For a more general overview. please refer to the Linux article for instructions on [how to install the latest Java 11 revision](linux.html).
 
 1. create a folder for Zulu 11 and make it the current folder. For instance, ``/opt/jdk``. You can use the commands
-```
-sudo mkdir /opt/jdk
-cd /opt/jdk
-```
+   ```shell
+   sudo mkdir /opt/jdk
+   cd /opt/jdk
+   ```
 2. verify the architecture of your device. You perform this issuing the command:
-```
-dpkg --print-architecture
-```
-this will show whether you need files for  Soft Float (```armsf```) or Hard Float (```armhf```) .
+   ```shell
+   dpkg --print-architecture
+   ```
+   this will show whether you need files for  Soft Float (```armsf```) or Hard Float (```armhf```) .
 Raspberry Pi 4 runs on ARM32-bit HF architecture (```armhf```), for instance. 
 
 
 Now, to install Zulu 11, you need to download and install a tar.gz package for your architecture following the steps described below (```armhf``` architecture assumed for reference):
 
 
 3. Download the latest Zulu 11 build from [the Azul download page](https://www.azul.com/downloads/zulu-community/?version=java-11-lts&architecture=arm-32-bit-hf&package=jdk) - at the time of writing, it's `` zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf ``. You can use the command:
-```
-sudo wget http://cdn.azul.com/zulu-embedded/bin/zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf.tar.gz
-```
+   ```shell
+   sudo wget http://cdn.azul.com/zulu-embedded/bin/zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf.tar.gz
+   ```
 4. Extract the archive
+    ```shell
+   sudo tar -xzvf -zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf
+   ```
+5. Install ```java``` and ```javac```
+   ```shell
+    sudo update-alternatives --install /usr/bin/java java /opt/jdk/zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf/bin/java 1
+    sudo update-alternatives --install /usr/bin/javac javac /opt/jdk/zulu11.43.100-ca-jdk11.0.9.1-linux_aarch32hf/bin/javac 1
+   ```
+6. change the current alternative:
+    ```shell
+    sudo update-alternatives --config java
+    ```
+   You'll be prompted to select the alternative, pick the zulu11 you installed.
+
+5. If you wish so, you can now delete the downloaded tar file
+    ```shell
+    sudo rm *.tar.gz
+    ```
+
+### Installation of openHAB:
+
```

Comment:
Please remove one blank line.

Reply:
do you mean 126? 125 is closing code..

Reply:
128 can be removed, since 129 is a blank line too.